sketcherMinimizer.cpp

```diff
@@ -1540,8 +1542,8 @@ vector<sketcherMinimizerPointF> sketcherMinimizer::shapeAroundLigand(int crownN)
     float distanceOfFirstCrown = 60;
     float distanceBetweenCrowns = 60;
     // find limits
-    auto atoms = _atoms;
-    auto bonds = _bonds;
+    auto atoms = m_atoms;
```

Comment:
is this _intending_ to make a copy?

      auto& atoms = m_atoms;

or even:

    const auto& atoms = m_atoms;

Reply:
yeah that works!